docs/index.md

```diff
@@ -1592,11 +1592,11 @@ Environment variables and default values:
 | `--vmwarevsphere-pool`            | `VSPHERE_POOL`            | -                        |
 | `--vmwarevsphere-compute-ip`      | `VSPHERE_COMPUTE_IP`      | -                        |
 
-## Base Operating Systems
-The Machine provisioning system supports several base operating systems.
-The default base operating system is Boot2Docker on local providers
-(VirtualBox, Fusion, Hyper-V, etc) and the latest Ubuntu LTS supported
-by the cloud provider.
+## Specify a Base Operating Systems
```

Comment:
@thaJeztah The headings are not using sentence cap in this whole file. Rather than make this one inconsistent, I left it as is for now.  I have to hit this file after 1.7 and tighten language anyway.